src/jit/SimdInl.h

```diff
@@ -311,3 +311,39 @@ static void emitGlobalSet128(sljit_compiler* compiler, Instruction* instr)
     sljit_emit_op1(compiler, SLJIT_MOV, SLJIT_R0, 0, SLJIT_MEM1(SLJIT_R0), context->globalsStart + globalSet->index() * sizeof(void*));
     sljit_emit_simd_mov(compiler, SLJIT_SIMD_STORE | SLJIT_SIMD_REG_128 | SLJIT_SIMD_ELEM_128, src.arg, SLJIT_MEM1(SLJIT_R0), JITFieldAccessor::globalValueOffset());
 }
+
+void emitSelect128(sljit_compiler* compiler, Instruction* instr, sljit_s32 jump_type)
+{
+    Operand* operands = instr->operands();
+    assert(instr->opcode() == ByteCode::SelectOpcode && instr->paramCount() == 3);
+
+    JITArg args[2];
+
+    const int m = sljit_has_cpu_feature(SLJIT_SIMD_REGS_ARE_PAIRS) ? 2 : 1;
+
+    JITArg target(operands + 3);
+    sljit_s32 dstReg = GET_TARGET_REG(target.arg, SLJIT_FR(2 * m));
+
+    simdOperandToArg(compiler, operands, args[0], SLJIT_SIMD_ELEM_128, SLJIT_FR(0 * m));
```

Comment:
This is always SLJIT_FR0.

Reply:
I guess that means the first `mov` can be removed.